src/geo/ui/infowindow.js

```diff
@@ -119,12 +118,15 @@ var Infowindow = View.extend({
         this.$('.js-content').css('max-height', this.model.get('maxHeight') + 'px');
       }
 
-      // If the template is 'cover-enabled', load the cover
+      this._renderLoader();
+      this._startLoader();
+
       this._loadCover();
 
       if (!this.isLoadingData()) {
         this.model.trigger('domready', this, this.$el);
         this.trigger('domready', this, this.$el);
+        this._stopLoader();
```

Comment:
Oh, it could happen that the infowindow doesn't contain a cover, and in that case we need to stop the loader. The stop should be in the trigger probably, but I didn't want to change the logic at this moment.